### PR TITLE
[QPE-573] infographic icons responsiveness

### PR DESCRIPTION
**Pull request by ahmed-Bazzara**
JIRA ticket
https://tretton37.atlassian.net/secure/RapidBoard.jspa?rapidView=33&projectKey=QPE&modal=detail&selectedIssue=QPE-573

### CODE REVIEW

**Comment by muenva** [comment]
After talking to @ahmed-Bazzara we concluded that it's only doable as a JS solution, the only missing piece is adding the padding of the segment column

File: src/statisticItem.js [hunk 1]
```diff
@@ -160,9 +185,12 @@ export default class StatisticItem extends Component {
       kpisRows,
       isShow
     };
-    const icon = (
-      <Icon isOnValue={true} valueIcon={valueIcon} iconSize={iconSize} value={numericValue} infographic={infographic} />
-    );
+    const icon = () => {
+      return (
+        <Icon ref="icon" columnNumber={columnNumber} isOnValue={true} valueIcon={valueIcon} iconSize={iconSize} value={numericValue} infographic={infographic} />
```
Comment (muenva) [inline src/statisticItem.js]: Why do we have check `isOnValue` inside the Icon component if it's set `true` here?

File: src/statisticItem.js [hunk 2]
```diff
@@ -65,6 +64,32 @@ export default class StatisticItem extends Component {
...
+    const iconPadding = 2;
+    if (wrapperElement && iconElement){
+      warpperWidth= wrapperElement.getBoundingClientRect().width;
+      iconWidth= iconElement.getBoundingClientRect().width + iconPadding;
+    }
+    let iconsColumnNumber = 20;
+    if ( iconWidth * iconsColumnNumber <= warpperWidth ){
+      iconsColumnNumber = 20;
+    }
+    else if (iconWidth * iconsColumnNumber > warpperWidth){
```
[the remark this answers is not in the document]
Comment (ahmed-Bazzara) [reply src/statisticItem.js]: agreed, this looks nicer

File: src/statisticItem.js (hunk 1 above)
Comment (ahmed-Bazzara) [reply src/statisticItem.js]: because in other cases, we send that prop as false/don't send it at all

Comment (muenva) [reply src/statisticItem.js]: okay, I see

**Pull request merged**